src/toolkitAPI.c

```diff
@@ -990,12 +992,33 @@ int DLLEXPORT swmm_getSubcatchStats(int index, TSubcatchStats *subcatchStats)
 		subcatchStats->precip *= (UCF(RAINDEPTH) / a);
 		// Cumulative Evaporation Volume
 		subcatchStats->evap *= (UCF(RAINDEPTH) / a);
+        
+        if (Nobjects[POLLUT] > 0)
+        {
+            for (p = 0; p < Nobjects[POLLUT]; p++)
+                subcatchStats->surfaceBuildup[p] *= Pollut[p].mcf;
```

Comment:
@katmratliff, i believe the problem is at this line.  I'll try to look into it later today but it looks like the units are being converted when I believe you already have it in lb (or kg) when you fetch these number from `subcatch_getBuildup(int j, int p)`.  I could be wrong! :-)

Reply:
ok! I can play around with it an test it out. I think you are probably correct, that makes sense... Thanks!

Reply:
@bemcdonnell unfortunately still the same result. I'll keep hunting through the unit conversion factors!

Reply:
I am following this discussion and the buildup from landuse_getBuildup is kg or lbs, so it starts out in kg or lbs

double  landuse_getBuildup(int i, int p, double area, double curb, double buildup,
                           double tStep)
//
//  Input:   i = land use index
//           p = pollutant index
//           area = land use area (ac or ha)
//           curb = land use curb length (users units)
//           buildup = current pollutant buildup (lbs or kg)
//           tStep = time increment for buildup (sec)
//  Output:  returns new buildup mass (lbs or kg)
//  Purpose: computes new pollutant buildup on a landuse after a time increment.
//
{
    int     n;                         // normalizer code
    double  days;                      // accumulated days of buildup
    double  perUnit;                   // normalizer value (area or curb length)

    // --- return current buildup if no buildup function or time increment
    if ( Landuse[i].buildupFunc[p].funcType == NO_BUILDUP || tStep == 0.0 )
    {
        return buildup;
    }

    // --- see what buildup is normalized to
    n = Landuse[i].buildupFunc[p].normalizer;
    perUnit = 1.0;
    if ( n == PER_AREA ) perUnit = area;
    if ( n == PER_CURB ) perUnit = curb;
    if ( perUnit == 0.0 ) return 0.0;

    // --- buildup determined by loading time series
    if ( Landuse[i].buildupFunc[p].funcType == EXTERNAL_BUILDUP )
    {
        return landuse_getExternalBuildup(i, p, buildup/perUnit, tStep) *
               perUnit;
    }

    // --- determine equivalent days of current buildup
    days = landuse_getBuildupDays(i, p, buildup/perUnit);

    // --- compute buildup after adding on time increment
    days += tStep / SECperDAY;
    return landuse_getBuildupMass(i, p, days) * perUnit;
}

Reply:
@bemcdonnell @dickinsonre thanks for the input and help, pollutant mass is being correctly reported now!